Scripts/RemediationScripts/Remediate-SetSQLServerMinReqTLSVersion.ps1

```diff
@@ -905,6 +913,13 @@ function Reset-SQLServerRequiredTLSVersion
            
             try
             {  
+                if([String]::IsNullOrWhiteSpace($minimalTlsVersionBefore))
+                {
+                    $sqlServersSkipped += $sqlServer
+                    Write-Host "The TLS version for the SQL Server: [$($_.ServerName)] can't be rolled back as it was set to null before remediation and TLS version can't be set to null." -ForegroundColor $([Constants]::MessageType.Warning)
+                    Write-Host $([Constants]::SingleDashLine)
+                    return;
```

Comment:
Wouldn't the "return;" here skip all remaining resources too?

Reply:
No, ForEach-Object works a little differently. Here return will work as continue.